[BUG] - Wrong Entrypoint in Dockerfile

### Describe the bug
When trying to run a docker image created by our Dockerfile the entrypoint path to Main class is wrong. Should be org.fungover.thunder.Main

### To Reproduce
Steps to reproduce the behavior:
1. docker build --tag thunder .
2. docker run thunder
3. Error: Could not find or load main class org.fungover.Main
Caused by: java.lang.ClassNotFoundException: org.fungover.Main


### Expected behavior
main method in class Main should be called.

--- a/Dockerfile
+++ b/Dockerfile
@@ -5,4 +5,4 @@ RUN mvn -f /src/pom.xml clean package
 FROM eclipse-temurin:21.0.2_13-jre-alpine
 COPY --from=build /src/target/*.jar /thunder.jar
 EXPOSE 1883
-ENTRYPOINT ["java", "-cp", "/thunder.jar", "org.fungover.Main"]
+ENTRYPOINT ["java", "-cp", "/thunder.jar", "org.fungover.thunder.Main"]

